src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -323,91 +336,314 @@ public static class PermutationFeatureImportanceExtensions
-        /// evaluated -- and it can use any dataset, not just the training set, to compute feature importance metrics.
-        /// </para>
-        /// <para>
-        /// PFI works by taking a labeled dataset, choosing a feature, and permuting the values
-        /// for that feature across all the examples, so that each example now has a random value for the feature and
-        /// the original values for all other features. The evalution metric (e.g. normalized mutual information) is then calculated
-        /// for this modified dataset, and the change in the evaluation metric from the original dataset is computed.
-        /// The larger the change in the evaluation metric, the more important the feature is to the model.
-        /// PFI works by performing this permutation analysis across all the features of a model, one after another.
-        /// </para>
-        /// <para>
-        /// In this implementation, PFI computes the change in all possible clustering evaluation metrics for each feature, and an
-        /// <code>ImmutableArray</code> of <code>ClusteringMetrics</code> objects is returned. See the sample below for an
-        /// example of working with these results to analyze the feature importance of a model.
-        /// </para>
-        /// </remarks>
-        /// <example>
-        /// <format type="text/markdown">
-        /// <![CDATA[
-        /// [!code-csharp[PFI](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance.cs)]
-        /// ]]>
-        /// </format>
-        /// </example>
-        /// <param name="ctx">The clustering context.</param>
-        /// <param name="model">The model to evaluate.</param>
-        /// <param name="data">The evaluation data set.</param>
-        /// <param name="label">Label column name.</param>
-        /// <param name="features">Feature column name.</param>
-        /// <param name="useFeatureWeightFilter">Use features weight to pre-filter features.</param>
-        /// <param name="topExamples">Limit the number of examples to evaluate on. null means examples (up to ~ 2 bln) from input will be used.</param>
-        /// <returns>Array of per-feature 'contributions' to the score.</returns>
-        public static ImmutableArray<ClusteringMetrics>
-            PermutationFeatureImportance(
-                this ClusteringContext ctx,
-                IPredictionTransformer<IPredictor> model,
-                IDataView data,
-                string label = DefaultColumnNames.Label,
-                string features = DefaultColumnNames.Features,
-                bool useFeatureWeightFilter = false,
-                int? topExamples = null)
+        public double Mean => _statistic.Mean;
+
+        /// <summary>
+        /// Get the standard deviation for the metric
+        /// </summary>
+        public double StandardDeviation => (_statistic.RawCount <= 2) ? 0 : _statistic.SampleStdDev;
```

Comment:
Why not calculate at n=2?

```suggestion
        public double StandardDeviation => (_statistic.RawCount <= 1) ? 0 : _statistic.SampleStdDev;
```

Reply:
You are right. Technically, it's a thing you can do, so I'll add it in.